setup.py

```diff
@@ -18,6 +18,14 @@ def ascii_bytes_from(path, *paths):
     return ascii_bytes
 
 
+def parse_requirements(filename):
+    """Load requirements from a pip requirements file."""
+    with open(filename) as f:
+        reqs = (req.strip() for req in f.readlines() if req and not req.startswith("#"))
+    f.close()
```

Comment:
Not needed, already closed here

Reply:
f*** ... i forgot to remove that statement 👍🏽